ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-02.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+    @State var titleInput: String = ""
+    @State var showDatePicker = false
+    @State var chooseDateTitle: String = "Choose date"
```

Comment:
This looks like a static string, are you sure we need it to be held as a state?

Reply:
Yes because I want to display a placeholder Choose date and when users choose their deadline it will be updated to the date chosen.